### PR TITLE
new resource link added to welcome page

### PR DESCRIPTION
**Pull request by MaddyDev**

### CODE REVIEW

File: extensions/resource-deployment/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "azdata": "^1.6.0"
   },
   "activationEvents": [
-    "onCommand:azdata.resource.create"
```
Comment (anthonydresser) [inline extensions/resource-deployment/package.json]: Is there a reason we are changing this activation event? I don't see anything about this PR that requires it.

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ export default () => `
 						<li><a href="command:registeredServers.addConnection">${escape(localize('welcomePage.newConnection', "New connection"))}</a></li>
 						<li><a href="command:workbench.action.files.newUntitledFile">${escape(localize('welcomePage.newQuery', "New query"))}</a></li>
 						<li><a href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
+						<li><a href="command:azdata.resource.create">${escape(localize('welcomePage.newResource', "New resource"))}</a></li>
```
Comment (kburtram) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: I thought we were going to come up with a better name for this command.  I don't think "New resource" makes sense in this context.

Comment (MaddyDev) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: this is just an initial commit for Alan to review the changes I made for adding to the welcome page. We'll make changes to it as we go.

Comment (kburtram) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: I'd recommend against making changes to the welcome screen until we've made a decision about basic things such as the name of the commands.  You could stick something in the command palette if you need a way to invoke the command, but I'd rather not add UI elements that are confusing, particularly if we think they may change.

Comment (MaddyDev) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: Sure Karl. These changes are on the deployment-workstream branch and not to master, so I took the liberty to make changes just to test if we can access external host content from welcome page. I'll not make any changes until we have made a decision.

**Pull request closed without merging**